source/lovelace/index.markdown

```diff
@@ -97,10 +97,40 @@ views:
 
 Now restart Home Assistant, navigate to `<YOUR HASS URL>/lovelace`. When you make changes to `ui-lovelace.yaml`, you don't have to restart Home Assistant or refresh the page. Just hit the refresh button at the top of the UI.
 
-## Custom Cards
+{% linkable_title Custom cards %}
 
 It is possible to add your own custom cards to show up in the Lovelace UI. For more information, check [the developer docs](https://developers.home-assistant.io/docs/en/lovelace_custom_card.html).
 
-## Current limitations
+{% linkable_title Current limitations %}
 
 This is the very very early version aimed at gathering feedback. Discussion and suggestions are welcome in the [ui-schema repository](https://github.com/home-assistant/ui-schema) and in the [chat](/join-chat/) in #lovelace.
+
+{% linkable_title Make Lovelace default %}
+
+Click the `>> Set lovelace as default page page on this device <<` in `dev-info` panel to make Lovelace the default interface when visiting `/`. 
+
+{% linkable_title Debugging %}
+
+To see what entities you left out, you can now use the menu `Unused entities` in the header of Lovelace,  the top right corner.
+
+<p class='img'>
+<img src='/images/lovelace/lovelace_unused_entites.png' alt='Screenshot of unused entities menu option'>
+Screenshot of unused entities menu option.
+</p>
+
+{% linkable_title Examples %}
+
+To get you started you can use the `demo` platfrom in your `configuration.yaml` and experiment with the various examples inside card docs.
+
+Other examples:
+- [ui-schema](https://github.com/home-assistant/ui-schema/blob/master/dev_repo_test_config) repo used by devs
+- [arsaboo](https://github.com/arsaboo/homeassistant-config/blob/master/ui-lovelace.yaml) own setup
+
+{% linkable_title Migration scripts %}
+
+Thare are two migration scripts:
+
+> The scripts might not be up to date with 0.73.0 on release day. Please be patient.
```

Comment:
remove the scripts, if they don't work with latest version. if they work remove the note